lib/themes/dosomething/paraneue_dosomething/Gruntfile.js

```diff
@@ -157,9 +157,11 @@ module.exports = function(grunt) {
           "src": [
             "js/**/*.js",
             "scss/**/*.scss",
-            "bower_components/neue/js/**/*.js",
-            "!bower_components/neue/js/modernizr/**/*.js",
-            "bower_components/neue/scss/**/*.scss"
+            "node_modules/dosomething-neue/js/**/*.js",
+            "!node_modules/dosomething-neue/js/modernizr/**/*.js",
```

Comment:
Yup, that excludes those files from the glob (since those are sources for additional custom tests, which are included blow).